### PR TITLE
doc(release): Release runbook updates

### PR DESCRIPTION
**Pull request by german-muzquiz**
Updates to how to release spin-cli and sponnet.

### CODE REVIEW

File: community/releases/release-manager-runbook/index.md [hunk 1]
```diff
@@ -269,13 +269,12 @@ channel to let them know that a new patch is available.
...
-    changes onto the Spin CLI release branch. As of writing, the Swagger
-    Codegen CLI uses 2.3.1; you can get that JAR [here](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar).
-    If using a different version, you can try modifying the version parameters
-    in the URL.
+    1. From the Gate repository, check out the release branch and generate the `swagger/swagger.json` file (it's not under source control):
+    ```
+    ./swagger/generate_swagger.sh
+    ```
+    
+    1. From the spin-cli repository, check out the release branch (release branches from gate and spin-cli must match) and follow the [instructions](https://github.com/spinnaker/spin/blob/master/CONTRIBUTING.md#updating-the-gate-api) in that repo to update the gate client. This involves creating and merging a PR to spin-cli release branch with the updated gate client.
```
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]: Can we be consistent with the casing of Spin CLI and Gate?

File: community/releases/release-manager-runbook/index.md [hunk 2]
```diff
@@ -295,9 +294,8 @@ channel to let them know that a new patch is available.
         - BOM_VERSION: This is the BOM to associate the Spin CLI release with. It is
         the latest Spinnaker release number, 1.xx.x.
 
-1. Publish a Sponnet minor version. Run the [publish.sh](https://github.com/spinnaker/sponnet/blob/master/publish.sh)
-script while passing in the version number of the new release.
-Example: VERSION="1.17.2" ./publish.sh
+1. Make a Sponnet [github release](https://github.com/spinnaker/sponnet/releases/new). Give it the same version as the newly released Spinnaker, with the tag prefixed with "v" (i.e. v${RELEASE}).
```
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
1. Make a Sponnet [GitHub release](https://github.com/spinnaker/sponnet/releases/new). Give it the same version as the newly released Spinnaker, with the tag prefixed with "v" (e.g., v1.22.0).
```

**Review by maggieneterval — APPROVED** [review]
thanks so much for updating this!

File: community/releases/release-manager-runbook/index.md (hunk 1 above)
Comment (dorbin) [reply community/releases/release-manager-runbook/index.md]: It looks like these are all references to the respective repositories, so we should use the repo names: "spin" and "gate"

File: community/releases/release-manager-runbook/index.md (hunk 2 above)
Comment (dorbin) [reply community/releases/release-manager-runbook/index.md]: Prefer "for example," rather than "e.g.," or "i.e.".

**Review by dorbin — APPROVED** [review]

**Review by dorbin — APPROVED** [review]

**Pull request merged**